README.md

```diff
@@ -66,6 +66,37 @@ evaluater = Evaluation(image_dir=dp.image_dir, job_dir=dp.job_dir)
 evaluater.run()
```

Comment:
chris, could you rename 'evaluater' into 'evaluator'?

Reply:
Done.